validator_test.go

```diff
@@ -5515,7 +5515,7 @@ func TestStructOnlyValidation(t *testing.T) {
 		LastName          string     `json:"lname"`
 		Age               uint8      `validate:"gte=0,lte=130"`
 		Number            string     `validate:"required,e164"`
-		AlternativeNumber string     `validate:"e164"`
+		AlternativeNumber string     `validate:"e164,omitempty"`
```

Comment:
This is NOT how omit empty work with validator. PLEASE read the documentation provided.

`omitempty` comes before whatever you want that to be true for.

Reply:
Where is it dritten, that `omitempty` needs to be before other validators?

```
Allows conditional validation, for example if a field is not set with a value (Determined by the "required" validator) then other validation such as min or max won't run, but if a value is set validation will run.
```